core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -100,6 +101,7 @@ object ZkSecurityMigrator extends Logging {
         false
       case _ =>
         CommandLineUtils.printUsageAndDie(opts.parser, usageMessage)
+        Exit.exit(1)
```

Comment:
Could you use the new `ToolsUtils.printUsageAndDie(...)` which you introduced?

Reply:
Sure. Thanks.